### DISCUSSION TITLE
Why are you ignoring ' fatal error: 'vpx/vp8.h' file not found', a bug reported in #905, #1533, #2189, #2165,

### DISCUSSION THREAD

**Discussion by p-baum** [rustdesk/rustdesk#3700] [General]
```
C:\Users\Paul\rustdesk\libs\scrap\vpx_ffi.h:1:10: fatal error: 'vpx/vp8.h' file not found
  thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ClangDiagnostic("C:\\Users\\Paul\\rustdesk\\libs\\scrap\\vpx_ffi.h:1:10: fatal error: 'vpx/vp8.h' file not found\n")', libs\scrap\build.rs:134:18
```

This is clearly a bug. You are ignoring because it is a build bug.  You are effectively declaring windows build are not supported. Perhaps other builds aswell.  I'm just trying on windows.

#905, #1533, #2189, #2165,

**Comment by HenrysCat** [comment]
The error message "'vpx/vp8.h' file not found" typically indicates that the build process for RustDesk, or a Rust dependency of RustDesk, is unable to find the VP8 codec header file. VP8 is a video codec used for WebRTC video streaming, and it's possible that RustDesk relies on this codec for some of its functionality.

To resolve this issue, you can try the following steps:

1. Install the required dependencies:

   Make sure you have all the necessary dependencies installed on your Windows system. In this case, you need to ensure that the VP8 codec library and header files are available. You can install them using a package manager like MSYS2 or manually download and build the VP8 codec.

   To install the VP8 codec library using MSYS2, follow these steps:

   a. Install MSYS2 by downloading the installer from the official website: https://www.msys2.org/

   b. Open an MSYS2 terminal and run the following command to update the package database and upgrade the installed packages:

      ```
      pacman -Syu
      ```

   c. Install the VP8 codec library by running:

      ```
      pacman -S mingw-w64-x86_64-libvpx
      ```

2. Set the appropriate environment variables:

   If you have manually installed the VP8 codec library or headers, you may need to set environment variables to help RustDesk's build system locate them. You can set the environment variables in your command prompt or in your build script.

   - Set `PKG_CONFIG_PATH` to the directory containing the `.pc` files for VP8. This typically includes the library and header paths. For example:

     ```
     export PKG_CONFIG_PATH=/path/to/libvpx/pkgconfig
     ```

   - Set `LIBVPX_SYS_STATIC` to ensure that the library is linked statically. This might be necessary for successful compilation. You can set it as an environment variable or in your build script:

     ```
     export LIBVPX_SYS_STATIC=1
     ```

3. Rebuild RustDesk:

After making sure that the required dependencies and environment variables are set, try to rebuild RustDesk. Navigate to the RustDesk source code directory and execute the build command again.

**Reply by voltagex** [reply]
Please don't add GPT generated stuff in here.

**Comment by voltagex** [comment]
@p-baum this error's from the build of scrap, which appears to be mostly unmaintained.

I hit this on Mac too. Do you have vcpkg installed?
